src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/PanelEditor.js

```diff
@@ -38,16 +44,22 @@ class PanelEditor extends PureComponent {
   };
 
   handleSave = () => {
-    const { toggleEditingOff } = this.props;
-
     this.setBreakpoint();
-    toggleEditingOff();
   };
 
   onEnter = () => {
     this.handleSave();
   };
 
+  onChange = async newOption => {
```

Comment:
This might get noisy if it keeps toggling states, but it's a good tradeoff since the immediate feedback's nice.

Reply:
yeah, unfortunately we need to go from error back to good on keystroke...